lib/crypto/tasks.py

```diff
@@ -26,12 +29,29 @@ def sign_addons(addon_ids, force=False, **kw):
     installs it.
     """
     log.info('[{0}] Signing addons.'.format(len(addon_ids)))
+
+    def file_supports_firefox(version):
+        """Return a Q object: files supporting at least a firefox version."""
+        return Q(version__apps__max__application=amo.FIREFOX.id,
+                 version__apps__max__version_int__gte=version_int(version))
+
+    is_default_compatible = Q(binary_components=False,
+                              strict_compatibility=False)
+    # We only want to sign files that are at least compatible with Firefox 10,
+    # or Firefox 37 if they are not default to compatible.
+    # The signing feature should be supported from Firefox 40 and above, but
+    # we're still signing some files that are a bit older just in case.
+    ff_version_filter = (
+        (is_default_compatible & file_supports_firefox('10')) |
```

Comment:
Maybe @kmaglione will be able to answer, but I think it's mainly to pick a version that is old enough, but not too old, because we don't want to sign most of the very old addons (that didn't get auto-bumped anyway).